services/jikan/controller.go

```diff
@@ -0,0 +1,89 @@
+package jikan
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/gin-contrib/cors"
+	"github.com/gin-gonic/gin"
+
+	"github.com/sdslabs/gasper/configs"
+	"github.com/sdslabs/gasper/lib/mongo"
+	"github.com/sdslabs/gasper/lib/utils"
+	"github.com/sdslabs/gasper/types"
+)
+
+// ServiceName is the name of the current microservice
+const ServiceName = types.Jikan
+
+func streamHandler(c *gin.Context) {
+	appName := c.Param("app")
+
+	metricsInterval, err := strconv.ParseInt(c.Query("interval"), 10, 64)
+	if err != nil {
+		metricsInterval = 2 * int64(configs.ServiceConfig.Mizu.MetricsInterval)
+	}
+
+	metricsCount, err := strconv.ParseInt(c.Query("count"), 10, 64)
+	if err != nil {
+		metricsCount = 10
+	}
+
+	chanStream := make(chan []types.M, 10)
```

Comment:
Having a large write timeout, I think the channel is no longer necessary

Lets keep it simple and toll back to the original implementation. Spawning an extra goroutine and channel for every request is costly